### PR TITLE
Content permissions final sweep

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
More concise language for Invite user

### CODE REVIEW

File: app/views/users/new/check.njk [hunk 1]
```diff
@@ -146,7 +146,7 @@
           },
           {
             key: {
-              text: "Organisations this user will have access to"
+              text: "Organisations this user will have access to:"
```
Comment (adamsilver) [inline app/views/users/new/check.njk:149]: Summary lists don’t have colons

File: app/views/users/new/check.njk [hunk 2]
```diff
@@ -163,7 +163,7 @@
           },
           {
             key: {
-              text: "For courses run and ratified by Somerset SCITT Consortium this user can"
+              text: "For courses run and ratified by Somerset SCITT Consortium this user can:"
```
Comment (adamsilver) [inline app/views/users/new/check.njk:166]: Summary lists don’t have colons

File: app/views/users/new/check.njk [hunk 3]
```diff
@@ -180,7 +180,7 @@
           },
           {
             key: {
-              text: "For courses run by Ventrus Teaching School Alliance and ratified by Somerset SCITT Consortium this user can"
+              text: "For courses run by Ventrus Teaching School Alliance and ratified by Somerset SCITT Consortium this user can:"
```
Comment (adamsilver) [inline app/views/users/new/check.njk:183]: Summary lists don’t have colons

File: app/views/users/show.njk [hunk 4]
```diff
@@ -154,7 +154,7 @@
           },
           {
             key: {
-              text: "Organisations this user has access to"
+              text: "Organisations this user has access to:"
```
Comment (adamsilver) [inline app/views/users/show.njk:157]: Summary lists don’t have colons

File: app/views/users/show.njk [hunk 5]
```diff
@@ -171,7 +171,7 @@
           },
           {
             key: {
-              text: "For courses run and ratified by Somerset SCITT Consortium this user can"
+              text: "For courses run and ratified by Somerset SCITT Consortium this user can:"
```
Comment (adamsilver) [inline app/views/users/show.njk:174]: Summary lists don’t have colons

File: app/views/users/show.njk [hunk 6]
```diff
@@ -188,7 +188,7 @@
           },
           {
             key: {
-              text: "For courses run by Ventrus Teaching School Alliance and ratified by Somerset SCITT Consortium this user can"
+              text: "For courses run by Ventrus Teaching School Alliance and ratified by Somerset SCITT Consortium this user can:"
```
Comment (adamsilver) [inline app/views/users/show.njk:191]: Summary lists don’t have colons

**Comment by laurahermionetennant92** [comment]
@adamsilver Happy for you to take out colons or I can if you haven't?

**Pull request merged**